### PR TITLE
Added Funding Requests - pending approval

### PR DESCRIPTION
**Pull request by Anynomouss**
The document outlines the most basic protocol for requesting Funding Request and their payments which will be 10% in Grin unless specifically requested otherwise.

### CODE REVIEW

File: funding_requests.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- a) Total amount of funding to be paid upon completion of the project
+- b) Total amount of funding, sub payments to be made upon completion of predefined milestones
+- c) Any other funding model if the applicant can provide good arguments to deviate from option a) and b)
+
+## Payments
+Payments will be made after reviewing the completed project or completed project milestones.
+Payments will by default be made 85% in BTC and 20% (15% base + 5% bonus) in Grin. 
+This effectively means applicants will be paid 105% of the amount they requested if they applicant choses to be paid in 15% or more in Grin.
+Payment in Grin will be based on a monthly average value while Bitcoin
```
Comment (hendi) [inline funding_requests.md]: > while Bitcoin ...?

File: funding_requests.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+We advise to make a funding request in the following format:
+
+- a) Total amount of funding to be paid upon completion of the project
+- b) Total amount of funding, sub payments to be made upon completion of predefined milestones
+- c) Any other funding model if the applicant can provide good arguments to deviate from option a) and b)
+
+## Payments
+Payments will be made after reviewing the completed project or completed project milestones.
+Payments will by default be made 85% in BTC and 20% (15% base + 5% bonus) in Grin. 
+This effectively means applicants will be paid 105% of the amount they requested if they applicant choses to be paid in 15% or more in Grin.
```
Comment (hendi) [inline funding_requests.md]: > they requested if they applicant choses 

the**y** applicant
cho**o**ses

File: funding_requests.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Payment in Grin will be based on a monthly average value while Bitcoin  
+In case an applicant prefers to be paid less than the default 15% +5% bonus payment in Grin, we expect sound argumentation why this deviation is required, such as personal circumstances.
```
Comment (hendi) [inline funding_requests.md]: Probably add that the +5% bonus is dependent on accepting >=15% payment in grin.

File: funding_requests.md (hunk 1 above)
Comment (Anynomouss) [reply funding_requests.md]: Exactly, you can add the same for Bitcoin, also one Month moving average.

Comment (hendi) [reply funding_requests.md]: Sorry, I was unspecific. I meant "while Bitcoin <END OF SENTENCE>", something is missing there.

**Review by future3OOO — APPROVED** [review]

**Review by MCM-Mike — APPROVED** [review]
Looks good to me

**Review by hendi — APPROVED** [review]

**Review by davidtavarez — APPROVED** [review]

**Pull request merged**